CIP-governance-metadata-cc-rationale/README.md

```diff
@@ -0,0 +1,210 @@
+---
+CIP: ?
+Title: Governance metadata - Constitutional Committee votes
+Category: Metadata
+Status: Proposed
+Authors:
+    - Ryan Williams <ryan.williams@intersectmbo.org>
+    - Eysteinh Magnus Hansen <eysteinsofus@gmail.com>
```

Comment:
```suggestion
    - Eystein Magnus Hansen <eysteinsofus@gmail.com>
```
First name is spelled differently here (https://github.com/cardano-foundation/CIPs/tree/master/CIP-1694#acknowledgements) & on Cardano Forum.

Reply:
@Willburn which is correct?!